lib/utils.py

```diff
@@ -45,3 +45,23 @@ def run_command(cmd, **kwargs):
                                         stdout=output, stderr=error_output)
 
     return output
+
+
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
```

Comment:
I've added Google-style param and return value doc to this function. I suggest we do the same for the rest of the code incrementally.